requiam/delta.py

```diff
@@ -33,66 +33,63 @@ def __init__(self, ldap_members, grouper_query_instance, batch_size,
         self.log.debug('entered')
 
         self.ldap_members = ldap_members
-        self.grouper_query_instance = grouper_query_instance
+        self.gq = grouper_query_instance
```

Comment:
The name `self.gq` is a bit cyiptic, I suggest using `grouper_qry` instead:
```suggestion
        self.grouper_qry = grouper_query_instance
```